ui-admin/src/App.tsx

```diff
@@ -81,7 +81,7 @@ function PageFrame() {
   return (
     <div className="d-flex">
       <AdminSidebar/>
-      <div className="flex-grow-1 d-flex flex-column" style={{ backgroundColor: '#ededed' }}>
+      <div className="flex-grow-1 d-flex flex-column" style={{ backgroundColor: '#fff' }}>
```

Comment:
I honestly don't have a strong opinion on this in terms of styling, but this change is pretty far-reaching and causes other parts of the admin app to look less defined. In some cases, it's a bit harder to visually associate actions with their context as a result. Just want to call it out to make sure that this is intended. For example:
![Screenshot 2023-09-28 at 4 28 53 PM](https://github.com/broadinstitute/juniper/assets/7257391/48adba53-fa44-4352-b759-0e137c3b19f4)

Reply:
ooh, that's a good catch, I hadn't tried that particular page -- I'll see if I can clean that up